tests/unit/s2n_tls13_compute_shared_secret_test.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "testlib/s2n_testlib.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <s2n.h>
+
+/* Just to get access to the static functions / variables we need to test */
```

Comment:
Which static function/variables are you referring to here?

Reply:
Good point.

Reply:
I believe you don't require the` #include "tls/s2n_tls13_handshake.c"` here, can we get rid of it?